validations/products.js

```diff
@@ -11,6 +11,13 @@ const addProductSchema = Joi.object().keys({
   }),
 });
 
+const getProductSchema = Joi.object().keys({
+  params: Joi.object({
+    id: Joi.string().required(),
+  }),
+});
+
```

Comment:
Heres how it should be : 
const getProductSchema= {
  params: Joi.object()
    .keys({
      proposalId: Joi.number().required(),
    })
    .unknown(false)
    .required(),
}
so try to test this . 
( set allowUnkown: false in validateSchema middleware)
if it works then modify the addProductSchema also